examples/slack-clone/nextjs-slack-clone-dotenvx/supabase/.env.local

```diff
@@ -0,0 +1,11 @@
+# .env.local
```

Comment:
Can we rename this file as `.env` because .local should be typically gitignored.